rules-tests/TypeDeclaration/Rector/Closure/UseRectorContainerConfiguratorRector/Fixture/fixture.php.inc

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\Closure\UseRectorContainerConfiguratorRector\Fixture;
+
+return static function (\Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator $containerConfigurator): void {
+};
+
+?>
+-----
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\Closure\UseRectorContainerConfiguratorRector\Fixture;
+
+return static function (\Rector\Core\DependencyInjection\Loader\Configurator\RectorContainerConfigurator $containerConfigurator): void {
+};
+
+?>
```

Comment:
Could you add a test fixture to skip the other non-rector configs?
We don't want to override Symfony `config/services.php` to Rector ones :) also ECS should be skipped.

Reply:
How it should be checked? IMHO there's no need for such fixtures since this rule is optional (I did not add it to any rule set). People who want opt-in and use it should just configure paths to ignore for this rule.

Reply:
Probably e2e test, that single registered configured rule did some job.